apps/graphql/src/dataloaders/bookingsLoader/BookingsLoader.js

```diff
@@ -18,17 +19,38 @@ const sanitizeBookings = (
   return bookings.map(booking => {
     const type = detectType(booking);
     const typeSpecificData = getTypeSpecificData(booking, type);
-
+    const passengers = sanitizePassengers(booking);
     return {
       bid: booking.bid,
       status: booking.status,
       passengerCount: booking.passengers.length,
+      contact: {
+        email: booking.contact.email,
+        phone: booking.contact.phone,
+      },
+      passengers,
       type,
       ...typeSpecificData,
     };
   });
 };
 
+const sanitizePassengers = (booking: BookingApiResult): Array<Passenger> =>
+  booking.passengers.map((passenger, index) => {
```

Comment:
Index is not used, can you please remove it?

We should update our eslint to catch these ones in the future.

Reply:
yes, missed that. Is removed now